docs/concepts/queries.md

```diff
@@ -59,6 +94,35 @@ The way that this component is setup would have some nice traits:
 2. When the request is re-triggered by the polling interval, it will add '...refetching' to the post name
 3. If a user closed this `PostDetail`, but then re-opened it within [the allowed time](../api/createApi#keepunuseddatafor), they would immediately be served a cached result and polling would resume with the previous behavior.
 
+### Selecting data from a query result
+
+Sometimes you may have a parent component that is subscribed to a query, and then in a child you may want to pick an item from that result without performing an additional query for a `getItemById` when you already have the result from a `getItems` query. `selectFromResult` allows you to select a specific segment from a result in a performant manner. When using `selectFromResult`, the hook will not rerender unless the underlying data of the selected item has changed and will ignore changes to other elements in a collection.
```

Comment:
Reading through this, I felt like the first sentence of this paragraph ran on longer than expected. Is there any way we could split this up into several smaller sentences?